docs/shared/spm-installation-panel.mdx

```diff
@@ -6,42 +6,46 @@ import {
 
 <ExpansionPanel title="Swift Package Manager Installation">
 
-**NOTE**: These instructions are intended for usage on Xcode 11 and higher. Xcode 11 is the first version of Xcode that integrates Swift Package manager and makes it _way_ easier to use than it was at the command line. If you are using older versions of Xcode, we recommend using CocoaPods.
+> **Note:** These instructions are for Xcode 11 or later. Xcode 11 is the first version of Xcode that integrates Swift Package manager, whereas older versions require using the command line. If you're using an older version of Xcode, we recommend using CocoaPods instead.
 
 <ExpansionPanelList>
 <ExpansionPanelListItem number="1">
 
 Go to **File > Swift Packages > Add Package Dependency...**
 
-![add a dependency](../source/screenshot/spm_packages_add_dependency.png)
+<img class="screenshot" src="../source/screenshot/spm_packages_add_dependency.png" alt="Adding an SPM dependency" />
 
 </ExpansionPanelListItem>
 <ExpansionPanelListItem number="2">
 
-Paste the URL to the Apollo iOS repo on GitHub ([https://github.com/apollographql/apollo-ios.git](https://github.com/apollographql/apollo-ios.git)) into the search bar, then hit the **Next** button: 
+Paste the URL to the Apollo iOS GitHub repo (`https://github.com/apollographql/apollo-ios.git`) into the search bar, then click **Next**:
 
-![paste in the url](../source/screenshot/spm_paste_url.png)
+<img class="screenshot" src="../source/screenshot/spm_paste_url.png" alt="Pasting the Apollo iOS GitHub URL" />
 
 </ExpansionPanelListItem>
 <ExpansionPanelListItem number="3">
 
-Select what version you want to use, then hit next. Xcode will automatically suggest the current version `Up to Next Major`, we **strongly** suggest that while the iOS SDK is on a `0.x.x` version scheme, you select `Up To Next Minor` instead, as we will still be releasing breaking changes on minor versions:
+Select which version you want to use, then click **Next**.
 
-![select a version](../source/screenshot/spm_select_version.png)
+> Xcode automatically suggests the current version, `Up to Next Major`. We **strongly** suggest that until the release of Apollo iOS `1.x`, you select `Up To Next Minor` instead, because we might release breaking changes on a minor version.
+
+<img class="screenshot" src="../source/screenshot/spm_select_version.png" alt="Selecting an Apollo iOS version" />
 
 </ExpansionPanelListItem>
 <ExpansionPanelListItem number="4">
-  
-Select which packages you want to use. If you're just getting started, try selecting just the main `Apollo` library first - you can always come back and add the other packages later if you need them. Then hit finish.
 
-![select the packages you want to use](../source/screenshot/spm_select_package.png)
+Select which packages you want to use, then click **Finish**.
+
+> If you're getting started, we recommend selecting just the main `Apollo` library for now. You can always add other packages later if you need them.
```

Comment:
What's the thought on making this a separate paragraph? I feel like it flows a little weird to tell people to click finish, then tell them what the recommended libs are.

Reply:
Good point, tweaked